web/beacon-app/cypress/e2e/TopicQuery/TopicQuery.ts

```diff
@@ -0,0 +1,243 @@
+import { And, Given, Then, When } from 'cypress-cucumber-preprocessor/steps';
+
+Given("I'm logged into Beacon", () => {
```

Comment:
This test is long and could get longer once we're able to test more scenarios. For this reason, I decided to keep it separate from the Topic Detail page test.